develop/Makefile

```
@@ -0,0 +1,25 @@
+DOCKER_NETWORK_NAME = kind
+
+-include ../.devcontainer/targets.mk
+
+LOCAL_IPV4_ADDRESS ?= $(shell hostname -I | awk '{print $$1}')
```

```suggestion
LOCAL_IP_ADDRESS ?= $(shell hostname -I | awk '{print $$1}')
```

Also, `hostname -I` is not available with BSD utils (e.g., on macOS) and the local IP won't be the IP used by the kind cluster if Docker is running in a VM, like with Docker Desktop on Mac. To fix this, we could use `host.docker.internal` if on a mac.

---

I'm don't know much about the MacOS setup, but we need an IP address because it is used with the [sslip.io](https://sslip.io/) to resolve different hostnames, e.g registry.192.168.0.2.sslip.io, spire.192.168.0.2.sslip.io,etc to the same istio ingress. We use the different hostnames to route to different services, in the case of spire and the fl-orchestrator we do that based on the TLS SNI. Using `host.docker.internal` would not work unfortunately because then sslip.io has no way to resolve that. We only need to expose these if we want to access the fl-suite from a fl-edge (similar to the one installed in the installation guide). What I can do is:
- Do not expose the services via sslip.io DNS.
- Assume that the fl-edge will also be running locally (like in the installation guide)
- And link via docker's networking the fl-edge with the fl-suite running in kind

I think then we will be able to overcome the Docker Desktop on Mac and Linux differences.

---

I'm not sure if `fl-suite` can than be accessed from an edge outside the local computer, as on Docker on Mac the containers are running in a VM. What does work though is [port forwarding through kind](https://kind.sigs.k8s.io/docs/user/ingress/). The forwarded port will then be on the host IP.